DLSv2/Core/ManagedVehicle.cs

```diff
@@ -127,16 +125,15 @@ public bool LightsOn
         /// </summary>
         public bool SirenOn { get; set; } = false;
         public Dictionary<string, int> SoundIds = new Dictionary<string, int>();
-        public Dictionary<string, (bool IsActive, int Index)> AudioControlGroups = new Dictionary<string, (bool, int)>();
-        public Dictionary<string, (bool IsManualing, int Index)> AudioCGManualing = new Dictionary<string, (bool, int)>();
-        public Dictionary<string, bool> AudioModes = new Dictionary<string, bool>();
-        public List<string> ActiveAudioModes = new List<string>();
+        public Dictionary<string, ControlGroupInstance<AudioControlGroup>> AudioControlGroups = new();
+        public Dictionary<string, ModeInstance<AudioMode>> AudioModes = new();
 
         // Registers input
         public void RegisterInputs()
         {
+            ControlsManager.ClearInputs();
```

Comment:
I still don't love re-registering all inputs every time the managed vehicle changes. See comment from DMs: 

> Registering/unregistering inputs every time you switch active vehicles is inefficient and confusing. I recommend registering all inputs from the INI once when the plugin loads. When the current "active" player-owned vehicle changes, unsubscribe all of the previous vehicle's event handlers and subscribe to all of the new vehicle's input event handlers, but don't re-load all inputs from scratch. Alternatively, never unsubscribe the event handlers, and just check the new bool for is it the player's active vehicle and return out of the handler if it isn't.